nailgun/entity_mixins.py

```diff
@@ -48,10 +48,18 @@
 class TaskTimedOutError(Exception):
     """Indicates that a task did not finish before the timout limit."""
 
+    def __index__(self, message, task_id):
```

Comment:
@martin-schlossarek Sorry for the delay in review, but is this supposed to be `__init__`?  Given the use in exception class instantiation within the PR, `__init__` would suffice, but I don't know if you have another intent for handling these exception classes.

Per documentation, `object.__index__()` must return an integer, and at the moment this returns nothing.